components/editor/utils/InlineUI.js

```diff
@@ -97,21 +97,23 @@ const InlineUI = ({ editor, node, isMatch, children }) => {
     }, 0)
   }
 
-  const isBreakout =
-    !parent(editor.state.value, node.key).type ||
-    parent(editor.state.value, node.key).nodes.size === 1
+  const currentParent = parent(editor.state.value, node.key)
+  const showMoveUI =
+    currentParent.type &&
+    currentParent.type === 'CENTER' &&
+    currentParent.nodes.size > 1
```

Comment:
is it still possible to jump to another center with this logic? Because a center always has a empty p at the end and therefore has two notes? What does this `nodes.size > 1` prevent?

Reply:
It's the same as the `isBreakout` logic it replaces but inverted, plus a check for the parent's type (CENTER) that I move from the infobox logic into here (your suggestion).
If it's confusing I can break it in two pieces again. 
And yes, you can still jump to another center.

Reply:
`nodes.size > 1` filters out breakout elements, where `nodes.size===1`

Reply:
What is an example for a breakout element? A figure group? Or do you mean edge 2 edge? Edge 2 edge are not children of center (won't fall into that logic) and figure groups are currently movable.

Reply:
Exactly, edge to edge elements. Basically i both want to not show the ui for:

1. edge to edge components
2. elements like infobox lists (which is the child of an infobox, not a center)

the `currentParent.type === 'CENTER'` is about elements like infobox lists (nothing to do with edge to edge, except that those shouldn't display the ui either)

Reply:
but I see what you mean, basically now I can remove the size condition